tests/test_model_api.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Author: USER
+Email: yli056@e.ntu.edu.sg
+Date: 10/14/2020
+"""
+
+
```

Comment:
better add a simple CLI test file

Reply:
Is the test still under construction so all of the code is commented?

Reply:
Nope, just notes for some main idea. You can work on it to 'translate' the curl to some test. You can also look into how to use a temp model weight file in GitHub action.